tests/browser/gaus/assign_gau_to_opportunity.robot

```diff
@@ -13,19 +13,18 @@ Assign GAU to Opportunity
     &{contact} =  API Create Contact    Email=skristem@robot.com
     &{opportunity} =  API Create Opportunity    &{Contact}[AccountId]    Donation    Name=Test GAU donation
     Go To Record Home  &{opportunity}[Id]
-    Load Related List    GAU Allocations
-    Select Related Dropdown    GAU Allocations
-    Wait For Locator    link-title    Manage Allocations
-    Click Link    link=Manage Allocations
+    Select Tab    Related
+    Click Special Related List Button    GAU Allocations    Manage Allocations
     Wait For Locator    frame    Manage Allocations
-    #Sleep    2
     Select Frame With Title    Manage Allocations
     Select Search    General Accounting Unit 0    &{gau1}[Name]
     Add GAU Allocation    Percent 0    50
     Click Button With Value    Add Row    
-    Select Search    General Accounting Unit 1    &{gau2}[Name] 
-    #sleep    2   
+    Select Search    General Accounting Unit 1    &{gau2}[Name]  
     Add GAU Allocation    Amount 1    20
-    Click Save    GAU
+    Click Button With Value    Save
+    Sleep    2
+    Select Window
+    Page Scroll To Locator    record.related.check_occurrence    GAU Allocations
     Verify Occurrence    GAU Allocations    2
```

Comment:
Looks like this one failed: https://mrbelvedereci.herokuapp.com/tests/result/31661067